ramls/external_patron.json

```diff
@@ -126,10 +94,9 @@
       },
       "minItems": 1,
       "maxItems": 3,
-      "uniqueItems": true,
-      "description": "Preferred email communication types"
+      "uniqueItems": true
     }
   },
-  "required": ["generalInfo", "address0", "contactInfo", "preferredEmailCommunication"],
+  "required": ["generalInfo", "address", "contactInfo"],
```

Comment:
For preferredEmailCommunication field, the expected minItems is 1. So anyhow this field will be required right?